less/index.less

```diff
@@ -64,113 +64,251 @@ h1, h2, h3, h4, h5 {
+            width: 100%;
+            display:flex;
+        }
+        display: flex;
+        margin-top: 72px;
+        justify-content: flex-end;
 
-header nav a {
-    text-decoration: none;
-    color: #b0b0b0;
-}
+        .cta-text {
+            @media (max-width:768px) {
+                width: 100%;
+                display:flex;
+                color: @primary-color;
+                font-size: 48px * 2;
+                margin: auto;
+            }
+            font-size: 72px;
+            text-align: center;
+            letter-spacing: 5px;
+            display: flex;
+            justify-content: center;
+            flex-flow: column;
+            align-items: center;
+            margin: 0 82px 0 0;
 
-header nav a:hover {
-    text-decoration: underline;
-    color: black;
-}
+            button {
+                border: 1px double black;
+                font-size: 16px;
+                background: white;
+                width: 170px;
+                height: 32px;
+                margin-top: 10px;
+                cursor: pointer;
 
-header .logo {
-    align-self: flex-end;
-    margin: 28px 0 0 91px;
-}
+                &:hover {
+                    background: black;
+                    color: white;
+                }
+            }
+        }
+        img {
+            @media (max-width:768px) {
+                display:none;
+            }
+        }
+    }
+    .main-content {
+        @media (max-width:768px) {
+            color: @third-color;
+        }
+        border-top: 2px solid black; 
+        border-bottom: 2px solid black;
+        margin: 75px 0 35px 0;
+        padding: 35px 0;
+        font-size: 16px;
+        line-height: 18px;
 
-/* Call To Action (CTA) Styles */
+        .top-content {
+            display: flex;
+            justify-content: space-around;
+        }
+        .middle-img {
+            @media (max-width:768px) {
+                max-width: 100%;
+            }
+            display: block;
+            margin: 30px auto;
+        }
+        .bottom-content {
+            display: flex;
+            justify-content: space-around;
 
-.cta {
-    display: flex;
-    margin-top: 72px;
-    justify-content: flex-end;
+            p {
+                padding-right: 20px;
+            }
+        }
+    }
+    .contact p {
+        @media (max-width:768px) {
+            color: @third-color;
+        }
+        margin: 15px 0;
+    }
+    footer {
+        @media (max-width:768px) {
+            color: negation(@secondary-color, @third-color); //Negation function (the opposite effect of difference creates a brighter color on the footer color
+        }
+        text-align: center;
+        margin: 40px 0 20px;
+    }
 }
 
-.cta .cta-text {
-    font-size: 72px;
-    text-align: center;
-    letter-spacing: 5px;
-    display: flex;
-    justify-content: center;
-    flex-flow: column;
-    align-items: center;
-    margin: 0 82px 0 0;
-}
+// Header Styles
```

Comment:
I would probably remove all these excess comments in the future (there's like 100 lines of them, it's just a bit much)